.eslintrc.yml

```diff
@@ -0,0 +1,87 @@
+root: true
+parser: typescript-eslint-parser
+plugins:
+  - typescript
+  - babel
+  - react
+  - prettier
+
+extends:
+  - prettier
+  - prettier/react
+
+rules:
+  ##########
+  # CUSTOM #
+  ##########
+  insecure-random: error
+
+  ###########
+  # PLUGINS #
+  ###########
+  # TYPESCRIPT
+  typescript/interface-name-prefix:
+    - error
+    - always
+  typescript/no-angle-bracket-type-assertion: error
+  typescript/explicit-member-accessibility: error
+  typescript/no-unused-vars: error
+  typescript/no-use-before-define:
+    - error
+    - functions: false
+      variables: false
+      typedefs: false
+  ## blocked by https://github.com/nzakas/eslint-plugin-typescript/pull/23
+  # typescript/member-ordering: error
+  ##
+  ## blocked by https://github.com/nzakas/eslint-plugin-typescript/issues/41
```

Comment:
Looks like this issue was closed, is this unblocked?

Reply:
For this one and the previous one, I’ve re-enabled these rules in my Episode II branch where I extract the configuration to another repo. I can re-enable these if you’d like, though.

Reply:
That's fine 👍